lib/social.ts

```diff
@@ -1,5 +1,8 @@
 import { Story } from '@prisma/client'
-import { storyCite } from '../components/stories/model'
+
+function storyCite({ displayName, postal }: Story) {
+  return displayName ? `${displayName} from ${postal}` : `From ${postal}`
+}
```

Comment:
I just copied this function from the old model.js for now, since we're not in a component context here, so we can't use a hook. Of course, this means that share content won't be translated. We'll need a bit of refactoring, but I didn't want to get into that now.